lib/jira/client/index.js

```diff
@@ -170,7 +170,7 @@ const batchedBulkUpdate = async (data, options, instance, installationId) => {
       },
     });
   });
-  await Promise.allSettled(batchedUpdates);
+  await Promise.all(batchedUpdates);
```

Comment:
Ooo this is an interesting change. This will immediately reject if any of the `batchedUpdates` fails, instead of waiting for all of them to complete (failing or not). Since we don't really care about the result either way I wonder if there is even a reason to `await` this call.

Reply:
Yeah probably there isn't. Happy to remove it.